spec/models/parent_object_spec.rb

```diff
@@ -235,4 +235,15 @@
       end
     end
   end
+
+  context "with correct visibility value" do
```

Comment:
We need to test for what happens for an empty visibility from the metadata cloud, and for a visibility other than the 3 we've defined in the model is given. Basically, should test the validation method for both nil and random strings.

Also, @orangewolf , should we make the visibilities a private method?